pkg/render/render_test.go

```diff
@@ -95,13 +95,13 @@ var _ = Describe("Rendering tests", func() {
 		// For this scenario, we expect the basic resources plus the following for Tigera Secure:
 		// - X Same as default config
 		// - 1 Service to expose calico/node metrics.
-		// - 1 ns (tigera-prometheus)
+		// - 2 ns (tigera-prometheus & tigera-dex)
```

Comment:
This comment isn't right as far as I can tell. I think `tigera-prometheus` is created by a manifest. I think the sum should be `(6 + 4 + 2 + 7 + 5 + 1 + 1) + 1 +1` Note the 2nd `+1` in the parenthesized section (it was added above but no here.